java-engine/build.gradle.kts

```diff
@@ -3,16 +3,13 @@ plugins {
     `maven-publish`
     signing
     id("com.diffplug.spotless") version "6.23.2"
-    id("io.github.gradle-nexus.publish-plugin").version("2.0.0")
+    id("io.github.gradle-nexus.publish-plugin") version "2.0.0"
     id("pl.allegro.tech.build.axion-release").version("1.16.0")
```

Comment:
Both with and without .() appears to be in use so should be fine, but not sure what the result of it is and why this change?

Reply:
This is the result of me desperately hacking a build file on a short time line because I want to close this project off today. They're the same line just different syntax. I don't mind changing it for consistency, feels fair... but won't you be a good citizen and suggest a change here so I can accept it? Yes I am super lazy, why do you ask?

Reply:
Cheers! Accepted